core/spec/models/spree/calculator/refunds/default_refund_amount_spec.rb

```diff
@@ -36,7 +36,7 @@
         order.adjustments.first.update_attributes(amount: adjustment_amount)
       end
 
-      it { is_expected.to eq line_item_price - (adjustment_amount.abs / line_item_quantity) }
+      it { is_expected.to be_within(0.1).of(line_item_price - (adjustment_amount.abs / line_item_quantity)) }
```

Comment:
Thanks for the PR.

Being within 0.1 (10 cents) is probably not good enough here. We should test for an exact amount.

This test passes both before and after the change made in `default_refund_amount.rb`. Is it possible to get a spec which clearly shows how the existing code was incorrect.